rpc/src/eth_rpc.rs

```diff
@@ -393,8 +380,13 @@ impl EthApiServer for KakarotEthRpc {
         _address: Address,
         _index: U256,
         _block_number: Option<BlockId>,
-    ) -> Result<H256> {
-        todo!()
+    ) -> Result<Bytes> {
```

Comment:
Nice catch! It seems reth's types are wrong. 
https://github.com/paradigmxyz/reth/blob/main/crates/rpc/rpc-api/src/eth.rs#L115

You can open an issue on their repo if you'd like to signal they're not respecting the ethereum json-rpc API https://ethereum.github.io/execution-apis/api-documentation/